week-8/Homework/mandatory/3-dog-photo-gallery/dogPhotos.js

```diff
@@ -0,0 +1,22 @@
+
+const ul = document.getElementById("saver");
+const btn = document.getElementById("click");
+const li = document.createElement("li");
+const myImg = document.createElement("img");
+
+function displayer(){
+    fetch("https://dog.ceo/api/breeds/image/random")
+    .then(response => response.json())
+    .then(data => data.message)
+    .then(function (img){
```

Comment:
Try to stick to the same function format (e.g. arrow functions) for consistency.